repository/hiqidashi_impl.go

```diff
@@ -145,7 +145,7 @@ func (repo *GormRepository) UpdateHiqidashiByID(ctx context.Context, hiqidashi *
 	hiqidashiMap["updated_at"] = hiqidashi.UpdatedAt
 
 	result := db.
-		Model(&hiqidashi).
+		Model(&model.Hiqidashi{}).Where("id = ?", hiqidashi.ID).
```

Comment:
ここ
```suggestion
		Model(&model.Hiqidashi{}).
		Where("id = ?", hiqidashi.ID).
		
```
って改行して見やすくしたいかも